### PR TITLE
modify ui for delete account

### LINKED ISSUE #685: modify ui for delete account
**Actual Behaviour**

Currently the 'Delete your account' text has theme color. Also, the dialog box is not responsive

**Expected Behaviour**

'Delete your account' should be red in color and the corresponding dialog box should be responsive.

**Screenshots of the issue**

image

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by AdityaSrivast**
Fixes #685 

Changes: 
- [x] Change 'Delete your account' color to red
- [x] Add hover effect for the button
- [x] Modify responsiveness of the delete account page

Surge Deployment Link: https://pr-688-fossasia-susi-accounts.surge.sh

Screenshots for the change:

image

### CODE REVIEW

File: src/components/Settings/Settings.css [hunk 1]
```diff
@@ -38,9 +38,12 @@ body{
 }
 
 .Link a {
-  color : #1DA1F5;
+  color : red;
```
Comment (AakashMallik) [inline src/components/Settings/Settings.css]: Please use hex value of **danger red**

File: src/components/Settings/Settings.css [hunk 2]
```diff
@@ -38,9 +38,12 @@ body{
...
   text-decoration : none;
 }
+.Link a:hover {
```
Comment (AakashMallik) [inline src/components/Settings/Settings.css]: Use material components from the view library. Don't add in effects like this rather use the buttons provided by the lib.

File: src/components/Auth/DeleteAccount/DeleteAccount.react.js [hunk 3]
```diff
@@ -219,7 +220,7 @@ class DeleteAccount extends Component {
           <StaticAppBar />
         </div>
         <div style={body}>
-          <Paper style={style} zDepth={5}>
+          <Paper style={style} className="delete-account" zDepth={5}>
```
Comment (AakashMallik) [inline src/components/Auth/DeleteAccount/DeleteAccount.react.js]: Don't use css, use the style object created for the component.

Comment (AdityaSrivast) [reply src/components/Auth/DeleteAccount/DeleteAccount.react.js]: @AakashMallik the css is used for responsiveness of the page. media queries are the best ways for the same. That's why I went for it.

**Comment by AakashMallik** [comment]
@samagragupta I agree...  bold looks odd there...

**Comment by AdityaSrivast** [comment]
@samagragupta @AakashMallik Should I make the font-weight normal for it?

**Comment by AakashMallik** [comment]
Yup..  I think normal font would be the right way to go about it.

**Comment by AdityaSrivast** [comment]
@AakashMallik @samagragupta I have made the changes. Can you please review again?

**Comment by AdityaSrivast** [comment]
@AakashMallik When logged in, https://pr-688-fossasia-susi-accounts.surge.sh/ will take you to the /settings route. You can refer that. It has nothing to do with the code. It's because of single page mechanism of surge.
Thanks

**Comment by AakashMallik** [comment]
> @AakashMallik When logged in, https://pr-688-fossasia-susi-accounts.surge.sh/ will take you to the /settings route. You can refer that. It has nothing to do with the code. It's because of single page mechanism of surge.
> Thanks

I don't get it. I logged in and the route broke. What is the single page mechanism you are talking about?

**Pull request merged**